dbfunction/users.js

```diff
@@ -0,0 +1,24 @@
+const User = require('../models/user');
+
+const UserDBClass = {
+  create: async (firstname, lastname, email, password) => {
+    const user = await User.create({
```

Comment:
anytime you declare a variable just to return the variable immediately, then it's always a good practice to return immediately. i:e, this code can be refactored to 

```
return  await User.create({
 
      firstname,
      lastname,
      email,
      password,
    }, {
      fields: ['firstname', 'lastname', 'email', 'password'],
    });

```